stdlib/asm/crypto/stark/constants.masm

```diff
@@ -64,40 +59,44 @@ const.DEEP_RAND_CC_PTR=4294912000
 #              .
 #       (FRI_COM_PTR + 128)     ---|
 #              .
-#              .                  | <- Remainder codeword and polynomial
+#              .                  | <- Remainder polynomial
 #              .
-#       (FRI_COM_PTR + 264-1)   ---|
+#       (FRI_COM_PTR + 256-1)   ---|
 #
 # For each FRI layer, we need 8 memory slots, one for storing the FRI layer commitment and one for
 # storing the word [a0, a1, log2(lde_size), lde_size] where a := (a0, a1) is the folding randomness
-# and lde_size is the size of the LDE domain. Since we are using a folding factor of 4 and the
-# maximal degree of the remainder polynomial that we allow is 7, an upper limit of 16 FRI layers is
-# ample and the number of memory slots we thus allocate for this is 128. Moreover, we allocate
-# an additional 128 slots for the remainder codeword and 8 for the remainder polynomial. These are
-# expected to be laid out right after the FRI commitments.
-# The total number of slots thus becomes 264.
+# and lde_size is the size of the LDE domain of the corresponding FRI layer.
+# Since we are using a folding factor of 4 and the maximal degree of the remainder polynomial
+# that we allow is 127, an upper limit of 16 FRI layers is ample and the number of memory slots
+# we thus allocate for this is 128. Moreover, we allocate an additional 128 slots for the remainder
+# polynomial which is expected to be laid out right after the FRI commitments.
 const.FRI_COM_PTR=4294912800
 
 # Commitment to main, auxiliary and composition polynomials traces
-const.MAIN_TRACE_COM_PTR=4294913200
-const.AUX_TRACE_COM_PTR=4294913204
-const.COMPOSITION_POLY_COM_PTR=4294913208
+const.MAIN_TRACE_COM_PTR=4294813200
+const.AUX_TRACE_COM_PTR=4294813204
+const.COMPOSITION_POLY_COM_PTR=4294813208
 
 # Instant-specific constants
-const.LDE_SIZE_PTR=4294913212
-const.Z_PTR=4294913216
-const.NUM_QUERIES_PTR=4294913220
-const.TRACE_LENGTH_PTR=4294913224
-const.TRACE_LENGTH_LOG_PTR=4294913228
-const.GRINDING_FACTOR_PTR=4294913232
+const.LDE_DOMAIN_INFO_PTR=4294813212
+const.LDE_DOMAIN_GEN_PTR=4294813213
+const.Z_PTR=4294813216
+const.NUM_QUERIES_PTR=4294813220
+const.REMAINDER_POLY_SIZE_PTR=4294813221
+const.REMAINDER_POLY_ADDRESS_PTR=4294813223
+const.NUM_FRI_LAYERS_PTR=4294813222
+const.TRACE_LENGTH_PTR=4294813224
+const.FRI_QUERIES_ADDRESS_PTR=4294813225
```

Comment:
nit: is there a reason for this to have suffix `_ADDRESS` rather than `_PTR`?

Reply:
This is because `REMAINDER_POLY_ADDRESS_PTR`, is a pointer to a pointer i.e., `REMAINDER_POLY_PTR` is stored at `REMAINDER_POLY_ADDRESS_PTR`